schemas/resource-schema.json

```diff
@@ -180,6 +180,25 @@
                   "minimum": 0
                 }
               }
+            },
+            "provider": {
```

Comment:
could it be needed for `error.resource` as well?

Reply:
Valid point indeed.